File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.*;

class Solution {
    public int[] solution(int l, int r) {
        ArrayList<Integer> list = new ArrayList<>();
        for(int i = l ; i <= r ; i++){
            String num = String.valueOf(i);
            boolean flag = true;
            for(int j = 0 ; j < num.length() ; j++){
                if(num.charAt(j) != '5' && num.charAt(j) != '0'){
                    flag = false;
                    break;
                }
            }
            if(flag) list.add(i);
        }
        if(list.isEmpty()) list.add(-1);
        return list.stream().mapToInt(Integer::intValue).toArray();
    }
}

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l,r+1):
        s = str(i)
        flag = True
        for ch in s:
            if ch not in ('0', '5'):
                flag = False
                break
        if flag:
            answer.append(i)
            
    return answer if answer else [-1]